packages/example-boards/src/utils/spread.ts

```diff
@@ -4,16 +4,43 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { OutputValues, code } from "@google-labs/breadboard";
+import { OutputValues, code as breadboardCode } from "@google-labs/breadboard";
+import { code } from "@google-labs/core-kit";
+import { CodeOutputConfig } from "../../../core-kit/dist/src/nodes/code";
+import { BreadboardType, JsonSerializable } from "@breadboard-ai/build/internal/type-system/type.js";
+import { Input } from "@breadboard-ai/build/internal/board/input.js";
+import { OutputPort } from "@breadboard-ai/build/internal/common/port.js";
 
-const spread = code<{ object: object }, OutputValues>((inputs) => {
+const spread = breadboardCode<{ object: object }, OutputValues>((inputs) => {
   const object = inputs.object;
   if (typeof object !== "object") {
     throw new Error(`object is of type ${typeof object} not object`);
   }
   return { ...object };
-});
+}); // TODO(Tina): This can be removed after boards switch to using `createSpreadNode` instead
 
 type spread = typeof spread;
 
 export { spread };
+
+export function createSpreadNode<T extends Record<string, BreadboardType | CodeOutputConfig>>(obj: Input<object & JsonSerializable> | OutputPort<JsonSerializable>, returnType: T) {
```

Comment:
Could you try replacing this with `unnest` from `core-kit`? It should do the same thing, and should handle types and JSON schemas automatically.

Reply:
I've removed this, and now use `cast` on the fetch response and use `unnest` to spread the value. Thank you!